main.swift

```diff
@@ -65,6 +65,7 @@ class StringReplacer {
     private let authorEmail: String
     private let gitHubURL: String
     private let year: String
+    private let today: String
```

Comment:
Also why does this need to be stored here? Can't it just be inline computed in `StringReplacer`?